main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
-:::
-
-## Platform: Linux Shell or Equivalent
+```sh
+corepack enable
+yarn --version # 1.x
+```
 
 The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Linux</a>, <a href="https://www.apple.com/macos/">MacOS</a>, and <a href="https://docs.microsoft.com/en-us/windows/wsl/">Windows Subsystem for Linux (WSL)</a>.
 
-- To open a terminal on MacOS, see **Applications>Utilities>terminal** in the **Finder**.
-- To launch a terminal on Linux, use the **Terminal** application.
-- To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
-
 ::: tip Mac Dev Tools
 On a Mac, you must first install
 [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
 :::
 
-## Install Go
+We support running a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/).
 
-Download Go from [go.dev/doc/install](https://go.dev/doc/install) and follow the instructions for your platform.
+Our user interfaces integrate with the [keplr](https://www.keplr.app/) wallet.
 
-```shell
-go version # Version 1.20.3 or higher
+## Create a project with the basic starter kit
+
+Choose a name such as `demo` and then:
+
+```sh
+yarn create @agoric/dapp demo
+cd demo
 ```
 
-## Install Node.js
+::: tip TODO: make dapp-game-places the default
 
-Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform.
-We recommend installing the LTS version of node 18.
+Until `dapp-game-places` is the default, use `--dapp-base` etc.
+to refer to it:
+
+```sh
+yarn create @agoric/dapp \
+  --dapp-base https://github.com/agoric-labs/ \
+  --dapp-template dapp-game-places \
+  demo
 
-```shell
-node --version # LTS version such as 18.16.0
 ```
 
-**Note:** Agoric will support all long-term support (LTS) versions of Node.js. 
+:::
 
-## Install the Yarn Package Manager
+### Project structure
 
-Follow [Yarn Installation](https://classic.yarnpkg.com/en/docs/install)
-instructions. For example:
+The `demo` directory now has:
 
-```shell
-npm install --global yarn
-yarn --version # 1.22.10 or higher
-```
+- **README.md**
+- **package.json** as usual for a JavaScript project, along with **yarn.lock**
+- **contract/** - smart contract source code, tests, etc.
+- **ui/** - web UI
+- **docker-compose.yml** - for running a local Agoric blockchain
 
-## Install Git
+::: tip TODO: prune api, agstate
 
-Follow [Git installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) or use your platform's package manager.
+Disregard these contents, for now:
 
-```shell
-npm install --global git
-git --version # 2.25.0 or higher
+- **\_agstate**
+- **api**
+  :::
+
+      node_modules
+
+## Install dependencies
+
+The UI depends on the React framework, and the contract depends on
+the Agoric framework. The packages in this project also have
+development dependencies for testing, code formatting, and static analysis.
+
+```sh
+yarn install  # may take several minutes
```

Comment:
I got the following error when I run `yarn install` following above instructions.

```
$ yarn install
yarn install v1.22.5
$ node -e "process.env.AGORIC_INSTALL && process.exit(0); console.warn('please use: agoric install . For details, see https://agoric.com/documentation/'); process.exit(1)"
please use: agoric install . For details, see https://agoric.com/documentation/
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
node --version
v18.12.1

yarn --version
1.22.19

Reply:
fixed in https://github.com/agoric-labs/dapp-game-places/commit/d96fb889061bf10f6293c2dcdf3032604109a7a0 , I think. try again?